packages/newsroom-signup/src/ApplyToTCR/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from "react";
+import { connect, DispatchProp } from "react-redux";
+import { BigNumber } from "bignumber.js";
+import { Civil, EthAddress, ListingWrapper, isInApplicationPhase } from "@joincivil/core";
+import { Parameters, getFormattedParameterValue } from "@joincivil/utils";
+import { getListing, getNewsroomMultisigBalance } from "../actionCreators";
+import ApplyToTCR from "./ApplyToTCR";
+import ApplyToTCRSuccess from "./ApplyToTCRSuccess";
+
+export interface ApplyToTCRStepOwnProps {
+  address?: EthAddress;
+  newsroom: any;
+  civil?: Civil;
+}
+
+export interface ApplyToTCRStepReduxProps {
+  userBalance: BigNumber;
+  multisigBalance: BigNumber;
+  applyStageLenDisplay?: string;
+  minDeposit: BigNumber;
+  multisigAddress?: EthAddress;
+  multisigHasMinDeposit: boolean;
+  listing?: ListingWrapper;
+}
+
+export type TApplyToTCRStepProps = ApplyToTCRStepOwnProps & ApplyToTCRStepReduxProps;
```

Comment:
Just curious, is the `T` a typo or shorthand for "type"?

Reply:
`T` is a shorthand for "type". I think I picked this up from a TS blog somewhere.